.gitignore

```diff
@@ -36,4 +36,16 @@ build.log
 interpolated-pom.xml
 
 # Temporary data ignored in camel-quarkus
-generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-hl7/id_file
\ No newline at end of file
+/generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-hl7/id_file
+/generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-js-dsl/src/main/resources/module.mjs
+
+# Temporary data ignored in quarkus-cxf tests
+/generated-platform-project/quarkus-cxf/integration-tests/quarkus-cxf-integration-test-ws-rm/derby.log
+
+# Workaround for https://github.com/quarkusio/quarkus/issues/33305
+/generated-platform-project/quarkus-cxf/integration-tests/Java2wsTest/
+/generated-platform-project/quarkus-camel/integration-tests/Java2wsTest/
+
+# Workaround for https://github.com/quarkiverse/quarkus-cxf/issues/900
+/generated-platform-project/quarkus-cxf/integration-tests/quarkus-cxf-integration-test-logging/src/main/resources/wsdl/CalculatorService.wsdl
+/generated-platform-project/quarkus-cxf/integration-tests/quarkus-cxf-integration-test-ws-security-client/src/main/resources/wsdl/dir/WssCalculatorService.wsdl
```

Comment:
Why do you generate temporary data in the source files? It looks like a bad practice to me.

(But thanks for fixing this as it was a bit annoying and I was wondering if I was doing something wrong)